src/components/shared/TooltipIfNeed.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Popup, Ref } from 'semantic-ui-react';
+
+import { selectors as settings } from '../../redux/modules/settings';
+import { isLanguageRtl } from '../../helpers/i18n-utils';
+
+/**
+ *
+ * @param props
+ * text: text that will be on the tooltip.
```

Comment:
Should we add such description for each hook we write?

Reply:
I was add it cause props of this component are not clear (part of props are child props)